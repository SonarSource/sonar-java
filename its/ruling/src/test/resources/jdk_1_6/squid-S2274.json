{
'jdk_1_6:java/awt/EventQueue.java':[
1050,
],
'jdk_1_6:java/util/concurrent/TimeUnit.java':[
292,
],
}
