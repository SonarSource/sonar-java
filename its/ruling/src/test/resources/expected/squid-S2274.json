{
'project:com/sun/corba/se/impl/naming/cosnaming/TransientNameServer.java':[
209,
],
'project:com/sun/corba/se/impl/oa/poa/POAManagerImpl.java':[
113,
],
'project:com/sun/corba/se/impl/orb/ORBImpl.java':[
1251,
],
'project:com/sun/corba/se/impl/orbutil/concurrent/CondVar.java':[
237,
298,
],
'project:com/sun/jmx/snmp/daemon/SnmpInformRequest.java':[
404,
],
'project:com/sun/jmx/snmp/daemon/SnmpQManager.java':[
193,
288,
],
'project:com/sun/jmx/snmp/daemon/SnmpSession.java':[
398,
],
'project:java/util/concurrent/ForkJoinTask.java':[
264,
],
'project:java/util/concurrent/TimeUnit.java':[
322,
],
}
